brave/browser-laptop
# Give more meaningful name to siteUtils method

clearSitesWithoutTags => clearHistory

Fixes https://github.com/brave/browser-laptop/issues/3639

Auditors: @bbondy @bridiver 

Test Plan:
- Visit about:history
- Use the "Clear browsing data button"
- Ensure "Browsing history" is on and click "OK"
- Go back to about:history
- Verify history is cleared
